extensions/expanded-opcodes/README.md

```diff
@@ -0,0 +1,87 @@
+# General design
+
+**Extension State: Under Development**  
+**Enabled by Default: Yes**  
+**Requires: Base, VWI**  
+**CPUID Bit: 9**
+
+# Overview
+
+This extension adds an expanded instruction format to allow for a larger number of opcodes to be used. It also includes a relative jump and call with larger displacements.
+
+# Added Instructions
+
+The following opcodes are now defined.
+
+| First byte    | Second Byte  | Third Byte   | Comment                                                                   |
+|:--------------|:-------------|:-------------|:--------------------------------------------------------------------------|
+| `111 00 000`  | `SS CCCCCC`  | `RRR RRR MM` | Register-Register operation                                               |
+| `111 00 ???`  | `SS CCCCCC`  | `RRR RRR MM` | When `???` is not `000`, reserved                                         |
+| `111 01 000`  | `SS CCCCCC`  | `RRR IIIII`  | Register-Immediate operation                                              |
+| `111 01 ???`  | `SS CCCCCC`  | `RRR IIIII`  | When `???` is not `000`, reserved                                         |
+| `111 10 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative jump                                                       |
+| `111 11 DDD`  | `DDDD DDDD`  | `DDDD DDDD`  | Large relative call if stack extension is implemented, otherwise reserved |
```

Comment:
Let's adopt the x86 terms for "short jump/call," "near jump/call," and "far jump/call." These would be "near"; but in x86 the near transfers get a 32-bit displacement which makes me think that this format should be adjusted to be more VWI than this. E.g. `1111 CCCC [4-bytes signed displacement]`.

Reply:
We could always make these use 2 instructions use additional bytes to make them far jumps.

Reply:
I think I've come up with a good solution to this. Take a look at the new commit